src/content/docs/de/guides/client-side-scripts.mdx

```diff
@@ -0,0 +1,237 @@
+---
+title: Skripte und Ereignisbehandlung
+description: >-
+  Hinzufügen von clientseitiger Interaktivität zu Astro-Komponenten mit nativen Browser
+  JavaScript-APIs.
+i18nReady: true
+---
+
+Du kannst deinen Astro-Komponenten Interaktivität hinzufügen, ohne ein UI-Framework (/de/core-concepts/framework-components/) wie React, Svelte, Vue usw. zu verwenden, indem du standardmäßige HTML-`<script>`-Tags verwendest. So kannst du JavaScript im Browser ausführen lassen und deinen Astro-Komponenten Funktionen hinzufügen.
+
+## Client-seitige Skripte
+
+Skripte können verwendet werden, um Event-Listener hinzuzufügen, Analysedaten zu senden, Animationen abzuspielen und alles andere, was JavaScript im Web ermöglicht.
+
+```astro
+// src/components/ConfettiButton.astro
+<button data-confetti-button>Zelebrieren!</button>
+
+<script>
+  // Importiere npm-Module.
+  import confetti from 'canvas-confetti';
+
+  // Finde unser Komponenten-DOM auf der Seite.
+  const buttons = document.querySelectorAll('[data-confetti-button]');
+
+  // Füge Event-Listener hinzu, um Konfetti zu werfen, wenn eine Schaltfläche angeklickt wird.
+  buttons.forEach((button) => {
+    button.addEventListener('click', () => confetti());
+  });
+</script>
+```
+
+Standardmäßig verarbeitet und bündelt Astro `<script>`-Tags und unterstützt damit den Import von npm-Modulen, das Schreiben von TypeScript und mehr.
+
+## Verwendung von `<script>` in Astro
+
+In `.astro`-Dateien kannst du clientseitiges JavaScript hinzufügen, indem du ein (oder mehrere) `<script>`-Tags hinzufügst.
+
+In diesem Beispiel wird durch das Hinzufügen der Komponente `<Hallo />` zu einer Seite eine Nachricht auf der Browserkonsole aufgezeichnet.
+
+```astro title="src/components/Hello.astro"
```

Comment:
In the line above you translated the component name, but didn't change the code sample title. You can either keep the component name in English or change this, it's up to you.
````suggestion
```astro title="src/components/Hallo.astro"
````